docs/en_US/install/wiiu.md

```diff
@@ -55,17 +55,41 @@ Bleeding edge builds have not been widely tested, and do not guarantee any stabi
 
 <img src="/assets/images/docs/install/wiiu/aroma/inkay-nightly-link.png" width="100%">
 
+## Nimble
+Nimble is a Aroma plugin that allows your console to recive Spotpass data again. Download the latest version from [Nimble's latest release](https://github.com/PretendoNetwork/Nimble/releases/latest) on GitHub and download the `30_nimble.rpx` file from the latest release.
```

Comment:
```suggestion
Nimble is a Aroma plugin that allows your console to recive SpotPass data again. Download the latest version from [Nimble's latest release](https://github.com/PretendoNetwork/Nimble/releases/latest) on GitHub and download the `30_nimble.rpx` file from the latest release.
```

Also this is a runon sentence. Can probably be paced better

Also Nimble is used for the policy list, not BOSS in general? BOSS content in games in general should still be working even without Nimble. @ashquarky Can you confirm this? Did Nintendo's shutdown somehow break BOSS content downloads in games?

Also CC @DaniElectra and @shutterbug2000 for input as well

Maybe we should look into trying to bundle Nimble into Inkay in this case? Nimble being separate made sense before when WWP was the only thing it really helped with, and not everyone cared about that, but if it's required for BOSS in general now that's a bigger issue